doc/api/dgram.md

```diff
@@ -261,7 +261,7 @@ changes:
                  and `length` parameters are optional now.
 -->
 
-* `msg` {Buffer|Uint8Array|string|array} Message to be sent
+* `msg` {Buffer|Uint8Array|string|array} Message to be sent.
```

Comment:
`Array`. might be good to specify the type of the members too if possible.

Reply:
>might be good to specify the type of the members too if possible.

Isn't that covered in the description below? If not happy to add info if you have any suggested wording.